docs/tutorials/google/spin_echoes.ipynb

```diff
@@ -0,0 +1,1347 @@
+{
```

Comment:
<p>It would be nice if this example circuit had a few more operations on the ancilla instead of directly a measurement.</p><p> </p><p>Though this problem is mitigated in the benchmark examples because of a different circuit generated by <code>sqrt_iswap</code> instead of <code>SYC</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4373/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>I think it's actually because the benchmark example circuit has 7 qubits whereas this circuit has 5 qubits - sometimes the optimizer removes the two-qubit gate on the ancilla and sometimes it doesn't.</p><p> </p><p>I could make this circuit be on 7 qubits also so that the two-qubit gate is not removed. (I just tested this and it is indeed not removed.) This would cause the rest of the circuits in this section and descriptions of what the optimizers do on the circuits to change, however. (This is the downside of having such explanations on non-trivial circuits.) What do you think?</p>

Reply:
If it's not too much pain and it's easy to add optimizer descriptions for the 7 qubit circuit, let's change it. Or else let's submit it. 

I don't have a strong preference. I leave the decision to you.

Reply:
<p>Quickly changed to this (well, 6 qubits). Cons: Fair bit of churn. Pros: Two-qubit gate is no longer removed and the example circuit looks less weird, and I can just do <code>circuit</code>  instead of <code>print(circuit.to_text_diagram(...)</code>  again, hopefully easier to review now :) </p><p> </p><p>Also note I changed the two-qubit gate from <code>cg.SYC</code>  to <code>cirq.ISWAP</code> so the inverse two-qubit gate in the circuit is clearer. </p>

Reply:
Thanks, this looks great!